lib/zipkin-tracer/tracer_factory.rb

```diff
@@ -1,3 +1,27 @@
+require 'zipkin-tracer/zipkin_tracer_base'
+
+module Trace
+  class ScribeTracer < ZipkinTracerBase
+    def initialize(options)
+      @scribe = options[:scribe]
+      super(options)
+    end
+
+    def flush!
+      @scribe.batch do
+        messages = spans.values.map do |span|
+          buf = ''
+          trans = Thrift::MemoryBufferTransport.new(buf)
+          oprot = Thrift::BinaryProtocol.new(trans)
+          span.to_thrift.write(oprot)
+          binary = Base64.encode64(buf).gsub("\n", "")
+          @scribe.log(binary, TRACER_CATEGORY)
```

Comment:
I think you want to make one call to log here.. it appears like this block makes a call for each span. scribe can accept a list of log entries in a single network call.